astroid/constants.py

```diff
@@ -0,0 +1,7 @@
+import sys
+
+PY36 = sys.version_info >= (3, 6)
```

Comment:
Isn't that always `True`? Python 3.6 is the min required version

Reply:
You're right, I removed the check for less than py36 but I can remove that too.